db/migrate/20170120164058_create_task_for_each_job_and_transfer_attributes.rb

```diff
@@ -9,6 +9,12 @@ class MiqTask < ActiveRecord::Base
   end
 
   def up
+    say_with_time("Deleting finished jobs older than 7 days") do
+      Job.where("updated_on < ?", Time.now.utc - 7.days).where("state = 'finished'").delete_all
```

Comment:
I prefer `7.days.ago.utc` to `Time.now.utc - 7.days`

Reply:
+1 on the `utc` there - I think it may work without, but there are strange edge cases